internal/controller/monitoring.coreos.com/prometheusrule_controller.go

```diff
@@ -116,14 +116,7 @@ func (r *PrometheusRuleReconciler) Reconcile(ctx context.Context, req ctrl.Reque
 		log.Info("PrometheusRule not found. Let's make one.")
 		prometheusRule, err = helpers.CreatePrometheusRule(slo, sli)
 		if err != nil {
-			err = utils.UpdateStatus(
-				ctx,
-				slo,
-				r.Client,
-				"Ready",
-				metav1.ConditionFalse,
-				"Failed to create Prometheus Rule",
-			)
```

Comment:
Stylistically and functionally disagree with this change, but I guess I don't care enough.

I wish gofmt would enforce this one as well

Reply:
Not sure I follow here